includes/Assets.php

```diff
@@ -94,11 +94,13 @@ public function get_asset_metadata( string $handle ): array {
 		// phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.UsingVariable
 		$chunks = is_readable( $chunks_file ) ? require $chunks_file : [];
 
-		$asset['dependencies'] = $asset['dependencies'] ?? [];
 		// A hash calculated based on the file content of the entry point bundle at <$handle>.js.
 		$asset['version'] = $asset['version'] ?? WEBSTORIES_VERSION;
-		$asset['js']      = $chunks['js'] ?? [];
-		$asset['css']     = $chunks['css'] ?? [];
+
+		$asset['dependencies'] = $asset['dependencies'] ?? [];
+		$asset['js']           = $chunks['js'] ?? [];
+		$asset['css']          = $chunks['css'] ?? [];
+		$asset['chunks']       = $chunks['chunks'] ?? [];
 
```

Comment:
These lines are moved just to tidy up the code right?

Reply:
Yes